### ISSUE TITLE
Address field performance is really bad

### ISSUE THREAD

**Issue by da2x** [brave/browser-laptop#6620]
I focus the address field, start typing, and then only the las third of what I typed shows up in the address field. This happens several times per day on both macOS and Fedora. This hasn’t really been a problem form me since I used IE6 on a really slow machine back in the day.

**Comment by Liunkae** [comment]
Possibly a regression of #2736 and #4360.

**Comment by bbondy** [comment]
Any difference with preview 9 of 0.13.0? Note you'll have to install on top after if you update to that with an official build.

**Comment by da2x** [comment]
@bbondy, the most recent released build is 0.13 preview 8 so I wouldn’t know anything about preview 9.

**Comment by bbondy** [comment]
I think I meant preview 8 but there are also some changes after that, so probably best to wait after all until preview 9.

**Comment by Liunkae** [comment]
Preview 9 is out.

**Comment by Liunkae** [comment]
+1 from #6713 (Keyboard input reading is delayed)

**Comment by bbondy** [comment]
Still a problem with the latest RC on the releases page?

**Comment by da2x** [comment]
I see no improvement in 0.13.0dev-RC5.

Test case:

1. Empty profile, no tabs, no cookies, no nothing.
2. Press Ctrl+N, and type google.com right  away

* Expected: window with google.com in the address field.
* Actual: e.com

More noticable with a new window, but the same problem persists when opening new tabs (Ctrl+T) or focusing the address field with Ctrl+L. There is a delay that feels like two full seconds before input is captured by the address field.

**Comment by dignifiedquire** [comment]
Seeing the same issue in the latest `master` builds

**Comment by Liunkae** [comment]
Any improvement with 0.14.0?

**Comment by onmyouji** [comment]
@Liunkae 

Performance is good when using new empty profile. But once I import my bookmark, the URL address field become slow. 

My laptop is quite old, but I don't have this problem when using other browser (e.g. Vivaldi).

**Comment by Liunkae** [comment]
Address bar performance with many bookmarks is expected to be fixed by 1.0.0 (#7453).

**Comment by ayumi** [comment]
URL bar performance has been improved dramatically in recent RC builds (fixes added a few days ago). Please try the next release (or an RC build!) and let me know how it works for you.

**Comment by alexwykoff** [comment]
Lack of commentary for followup, assuming fixed.

**Comment by The-Clockmaker** [comment]
This is still an issue on my ubuntu version, I'm using the latest stable version

**Comment by luixxiul** [comment]
@The-Clockmaker hi, would you mind providing us steps tor reproduce the issue? thanks!

**Comment by The-Clockmaker** [comment]
Yea sure, although I have noticed that with hardware acceleration disabled things are quite a bit faster, tho still slow.

OS:
elementary OS 0.3.2 Freya which is built on Ubuntu 14.04

Specs:
Intel i5-2410M Quad Core 2.30GHz
6GB ram
240 gb ssd
nvidia GeForce GT 540M 1GB

Brave version info:
Name	Version
Brave	0.17.16
rev	2d2ee2f
Muon	4.1.8
libchromiumcontent	59.0.3071.115
V8	5.9.211.38
Node.js	7.9.0
Update Channel	dev
os.platform	linux
os.release	4.4.0-83-generic
os.arch	x64

Booting Google Chrome: 2 seconds.
Booting Mozilla Firefox: 3 seconds.
Booting Brave Browser wihout hardware acceleration: 14 seconds.
Booting Brave Browser with hardware acceleration enabled: 39 seconds.

With for example 2 tabs open, when clicking a link on youtube, it will take 4 seconds before it starts loading the page.
There are cpu spikes sometimes up to 70% when I'm on youtube, google image search, outlook, for example.
When these cpu spikes occur, opening a new tab can take ages.
Then when hovering over another tab with say 4 tabs open, it can take up to 6 or 7 seconds before it's actually highlighted (I have tab preview disabled, because then it was even worse).
Typing in the url bar can also take about 2 to 3 seconds before it responds to what I have typed.

Also, the menu bar keeps showing up every time Brave is restarted, even though in settings I've set it to always hide.

None of these issues exist in either Chrome, Firefox or Midori, so it must be a Brave Browser issue, which is really too bad because I love the design, it's just nearly unusable for me.

Ps. By latest stable version I meant Brave
